sdkproject/Assets/Mapbox/Unity/MeshGeneration/LayerVisualizers/VectorLayerVisualizer.cs

```diff
@@ -317,6 +459,18 @@ private bool IsFeatureValid(VectorFeatureUnity feature)
 			return true;
 		}
 
+		protected void PreProcessFeatures(VectorFeatureUnity feature, UnityTile tile, GameObject parent)
+		{
+			////find any replacement criteria and assign them
+			foreach (var goModifier in _defaultStack.GoModifiers)
```

Comment:
This approach works well for now, but, it breaks the general architecture where `VectorLayerVisualizer` is bypassing the ModifierStack to communicate directly with GoModifiers. It's bad design and we should get rid of `ModifierStack` or move this part of the code to `ModifierStack` in the future.